### ISSUE TITLE
No additional examples

### ISSUE THREAD

**Issue by RStrydom** [milligram/milligram#50]
Under each section in the documentation is a link that says "See more examples of {section} here.". In some of these sections multiple implantations are discussed but not showcased, for instance the Codes section. Please can we either provide more examples or remove the link as it is quite frustrating to the user: landing on a page showing an isolated view of the section you just clicked on, on its own page.

**Comment by tomholford** [comment]
Possible to add the example pages to source control? I forked with the intention of building them out but couldn't find them :)

**Comment by olorton** [comment]
@tomholford I believe you will find the examples here: https://github.com/milligram/milligram.github.io

**Comment by tomholford** [comment]
Ah of course, thanks @olorton. Added an example :)

**Comment by cjpatoilo** [comment]
Hello guys.
Recently I updated the repository and I created two account, in [Codepen](https://codepen.io/milligram) and another in [Codepad](https://codepad.co/milligram), to support new examples. In the future I want to create a section for use cases and add more examples. I appreciate the interest in improvements and who can help with tips and examples, it will be amazing. For now I will closed this issue. 

Anyway sorry for the delay, I was out for few time, but this month will update all repositories and website.
